HuliWebShop/src/main/java/huli/example/huliwebshop/services/OrderServiceImpl.java

```diff
@@ -32,21 +33,21 @@ public Long createOrder(OrderDTO orderDTO) {
             throw new IllegalArgumentException("User not found with ID: " + orderDTO.getUserId());
```

Comment:
getting the current user from the dto is a huge security risk: if i (as a malicious actor) manage to find out the id of a user, i could create orders in their name.